### ISSUE TITLE
How to make two files into one bittorrent seed

### ISSUE THREAD

**Issue by wnark** [fbdtemme/torrenttools#28]
For example like this
```
root@ubuntu:/home/ubuntu# torrenttools create "/var/filebrowser/srv/VMware-workstation-full-16.2.1-18811642.exe /var/filebrowser/srv/VMware-workstation-full-16.2.1-18811642.exe.md5" -a "https://tracker.tamersunion.org/announce" "udp://opentor.org:2710/announce" -d "router.utorrent.com:6881" "router.bittorrent.com:6881" "dht.transmissionbt.com:6881" "router.bitcomet.com:6881" "dht.aelitis.com:6881" -o "VMware-workstation-full-16.2.1-18811642.exe.torrent"
Error: Path does not exist: /var/filebrowser/srv/VMware-workstation-full-16.2.1-18811642.exe /var/filebrowser/srv/VMware-workstation-full-16.2.1-18811642.exe.md5
```
I want to package and distribute the file with its md5 file, but it looks like it cannot be created at this stage

**Comment by wnark** [comment]
Because I store files in a directory structure similar to a mirror station, it is not convenient to package them into folders.

**Comment by wnark** [comment]
![image](https://user-images.githubusercontent.com/24355466/145921444-80aa89d6-c3ff-4ee9-b194-d4053893ab67.png)
Does the --include parameter here provide this function, but your demo here seems to be wrong

**Comment by wnark** [comment]
`torrenttools create /var/filebrowser/srv  --include "/var/filebrowser/srv/corona_3.09A.zst.md5" -a "https://tracker.tamersunion.org/announce" "udp://opentor.org:2710/announce" -w http://192.168.149.133:7000/ -d "router.utorrent.com:6881" "router.bittorrent.com:6881" "dht.transmissionbt.com:6881" "router.bitcomet.com:6881" "dht.aelitis.com:6881" -o "/var/filebrowser/srv/corona_3.09A.zst.torrent"`

Understood, "--include" matches the path, not the file name, so you need to use the complete directory structure under the "create" directory

**Comment by wnark** [comment]
`
torrenttools create /var/filebrowser/srv  --include "/var/filebrowser/srv/corona_3.09A.zst$" "/var/filebrowser/srv/corona_3.09A.zst.md5$" -a "https://tracker.tamersunion.org/announce" "udp://opentor.org:2710/announce" -w http://192.168.149.133:7000/ -d "router.utorrent.com:6881" "router.bittorrent.com:6881" "dht.transmissionbt.com:6881" "router.bitcomet.com:6881" "dht.aelitis.com:6881" -o "/var/filebrowser/srv/corona_3.09A.zst.torrent"`

If you need to match a file exactly, you need to use an absolute path to match

**Comment by wnark** [comment]
Your matching here is to traverse the directory, record the paths of all files, and then perform regular matching?

**Comment by fbdtemme** [comment]
> Your matching here is to traverse the directory, record the paths of all files, and then perform regular matching?

Correct. Current matching is agains the absolute path of the file which is somewhat unintuitive.

This should be changed to match the path of the files relative to the main root directory passed to `torrenttools create`.
So then you could simply pass:

```
torrenttools create /var/filebrowser/srv --include "corona_3.09A.zst$" "corona_3.09A.zst.md5$"
```

Thanks for reporting, I will fix this in the next release.
